src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 Coffee Client, 0x150 and contributors.
+ * Some rights reserved, refer to LICENSE file.
+ */
+
+package coffee.client.feature.module.impl.combat;
+
+import coffee.client.CoffeeMain;
+import coffee.client.feature.module.Module;
+import coffee.client.feature.module.ModuleType;
+import coffee.client.helper.event.EventType;
+import coffee.client.helper.event.Events;
+import coffee.client.helper.event.events.PacketEvent;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.network.packet.c2s.play.PlayerInteractEntityC2SPacket;
+import net.minecraft.network.packet.c2s.play.ClientCommandC2SPacket;
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket;
+import net.minecraft.util.math.Vec3d;
+
+import java.util.Objects;
+
+public class KnockbackPlus extends Module {
+
+
+    public KnockbackPlus() {
+        super("KnockbackPlus", "Makes you give the thing you're hitting extra knockback", ModuleType.COMBAT);
+        Events.registerEventHandler(EventType.PACKET_SEND, event1 -> {
```

Comment:
Use the more modern event listener, this shit is legacy

Reply:
i'm still very very new to this stuff, but i'll try using that and get back to this

Reply:
yeah i'm gonna be real i dunno what you're talking about

Reply:
the `@EventListener` annotation shit, look in any other module on how to use it

Reply:
alright, did that, should be good now